examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -1112,6 +1117,9 @@ void setup(void) {
   irsend.begin();
   offset = irsend.calibrate();
 #if IR_RX
+#if IR_RX_PULLUP
+  pinMode(IR_RX, INPUT_PULLUP);
```

Comment:
(No action required) Hmm. We should probably add that as an option in the library.

Reply:
Sure, that will even better. I was afraid to break existing APIs.